src/librustdoc/html/render/cache.rs

```diff
@@ -360,6 +366,7 @@ crate fn get_real_types<'tcx>(
             insert_ty(res, tcx, arg.clone(), ty_generics);
         }
     } else {
+        // This is not a type parameter so we can add it directly (and look for its trait bounds).
```

Comment:
Do things that aren’t type parameters have trait bounds, or are we talking about its own nested type parameters.

Reply:
Oh damn, I was talking about its own nested type parameters. Good catch!